lib/DeviceHive.js

```diff
@@ -332,18 +368,18 @@ class DeviceHive{
    * @returns 
    * @memberof DeviceHive
    */
-  subscribeCommands(deviceIds, subscriber, commandFilter){
+  subscribeCommands(deviceIds, subscriber, commandFilter) {
     commandFilter.deviceIds = deviceIds;
-    if (getType() === `ht`){
+    if (api.getType() === 'http') {
```

Comment:
Why not encapsulate the protocol checking into the API class and make all api calls like in "proxy" or "adapter" style?
Probably it would be better to extend DeviceHive with API class to decrease amount of wrapper functions.

Reply:
Yes, will make changes in api, ws, http services today, have some more ideas.